Doc/library/stdtypes.rst

```diff
@@ -2547,11 +2547,11 @@ arbitrary binary data.
             bytearray.join(iterable)
 
    Return a bytes or bytearray object which is the concatenation of the
-   binary data sequences in the :term:`iterable` *iterable*.  A
-   :exc:`TypeError` will be raised if there are any values in *iterable*
-   that are not :term:`bytes-like objects <bytes-like object>`, including
-   :class:`str` objects.  The separator between elements is the contents
-   of the bytes or bytearray object providing this method.
+   binary data sequences in the *iterable*.  A :exc:`TypeError` will be
```

Comment:
@berkerpeksag  I don't remember any specific reason for that. Should I include `the` in `str.join()` documentation?